cli/src/test/java/io/specmesh/cli/ProvisionFunctionalTest.java

```diff
@@ -64,29 +62,6 @@ void tearDown() {
         admin.close();
     }
 
-    @Test
-    void shouldProvisionCmdWithLombock() {
-        final var provision = Provision.builder().brokerUrl("borker").aclDisabled(false).build();
-        assertThat(provision.aclDisabled(), is(false));
-        assertThat(provision.brokerUrl(), is("borker"));
-    }
-
-    @Test
-    void shouldNotSwallowExceptions() {
```

Comment:
This test is moved to `TopicProvisionerTest` as it has nothing to do with `Provision`.